cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -185,3 +189,69 @@ func (w *wizard) callReward() {
 	fmt.Printf("Calling reward for round %v\n", c)
 	httpGet(fmt.Sprintf("http://%v:%v/reward", w.host, w.httpPort))
 }
+
+func (w *wizard) vote() {
+	if w.offchain {
+		glog.Error("Can not vote in 'offchain' mode")
+		return
+	}
+
+	fmt.Print("Enter the contract address for the poll you want to vote in")
+	poll := w.readStringAndValidate(func(in string) (string, error) {
+		if !ethcommon.IsHexAddress(in) {
+			return "", fmt.Errorf("invalid hex address address=%v", in)
+		}
+		return in, nil
+	})
+
+	var (
+		confirm = "n"
+		choice  = types.VoteChoice(-1)
+	)
+
+	for confirm == "n" {
+		choice = types.VoteChoice(-1)
+		choices := w.showVoteChoices()
+		if choices == nil {
+			return
+		}
+
+		for ok := choice.IsValid(); !ok; {
+			fmt.Printf("Enter the ID of the option you want to vote for")
```

Comment:
"option" -> "choice"

Reply:
So in my mind you choose an option, only after you have chosen does the option become a choice but I don't wanna discuss semantics here so I'm fine renaming everything to choice. It just felt like weird use of the word here to me.

Reply:
fixed in 2857efe